sources/packages/backend/apps/queue-consumers/src/services/student-file/student-file.service.ts

```diff
@@ -49,7 +55,14 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
         UNABLE_TO_SCAN_FILE,
       );
     }
-    await this.updateFileScanStatus(studentFile.uniqueFileName, isInfected);
+    if (isInfected) {
+      processSummary.warn("Virus found.");
+      processSummary.info("Updating infected file name and status.");
+      await this.updateInfectedFile(studentFile);
```

Comment:
It seems not clear to me to have the method `updateInfectedFile` and also the `updateFileScanStatus`.
Maybe the method `updateFileScanStatus` can be responsible for calling the method `updateInfectedFile` of the ``updateInfectedFile` logic can be moved inside the `updateFileScanStatus`.